test/demo/utils/common.ts

```diff
@@ -0,0 +1,110 @@
+import {Identity} from 'openfin/_v2/main';
+
+import {Intent} from '../../../src/client/intents';
+import {testManagerIdentity} from '../constants';
+
+import {fin} from './fin';
+import * as fdc3Remote from './fdc3RemoteExecution';
+import {delay} from './delay';
+
+export type Boxed<T> = {value: T}
+
+/**
+ * Identity with a mandatory `name`
+ */
+export interface AppIdentity extends Identity {
```

Comment:
This name may cause some confusion as strictly speaking an app in OF terms is only a uuid whereas a uuid/name pair is an OF window identifier. This is really more of a problem of consolidating OF nomenclature with FDC3, but maybe we could add some clarifier to this type to make it clear what it's an identifier for.

Reply:
Yeah, this was a big conversation with Phil while you were away 😛. Basically the way we declare the constants holding data about our test apps was kinda odd, because sometimes name was being treated as an app directory name, and sometimes name was being treated as an Identity name, and putting a type to this data highlighted that,

I've renamed the type to reflect that it's kinda fuzzy what this type really _is_ and added a comment about the use of `name`

Reply:
Not the first time it's been talked about and certainly won't be the last. As long as it's mostly clear from context what it means then I think we should be fine.